src/cosl/coordinated_workers/worker.py

```diff
@@ -394,7 +395,13 @@ def _add_readiness_check(self, new_layer: Layer):
             return
 
         new_layer.checks["ready"] = Check(
-            "ready", {"override": "replace", "http": {"url": self._readiness_check_endpoint(self)}}
+            "ready",
+            {
+                "override": "replace",
+                # threshold gets added automatically by pebble
+                "threshold": 3,
```

Comment:
I was convinced 3 was the default value, but either way, good to make it explicit

Reply:
yes, but we compare dicts, so we should be explicit